src/Mod/TechDraw/Gui/QGIViewDimension.cpp

```diff
@@ -351,21 +351,30 @@ void QGIDatumLabel::hoverLeaveEvent(QGraphicsSceneHoverEvent* event)
 
 QRectF QGIDatumLabel::boundingRect() const
 {
-    return childrenBoundingRect();
+    QRectF totalRect;
+    for (QGraphicsItem* item : m_textItems->childItems()) {
+        if (QGCustomText* customText = dynamic_cast<QGCustomText*>(item)) {
```

Comment:
I'd use auto here to avoid specifying the type twice on the line:
```suggestion
        if (auto* customText = dynamic_cast<QGCustomText*>(item)) {
```

Reply:
Incorporated.